app/managers.py

```diff
@@ -0,0 +1,60 @@
+import sqlite3
+

```

Comment:
should be blank line between different types of imports - os and sqlite3 are build-in so they are the same type. only from app.models import Actor should be separated